Manager/BackupManager.php

```diff
@@ -71,17 +71,8 @@ public function execute()
             // write log
             $this->logger->critical($e);
 
+            //Should we throw a general exception here? Maybe we should return a bool.
```

Comment:
It seams irresponsible to Catch everything and just rethrow it... Or is it? => Agree

But, if someone run the command, and an error is thrown but remains silent, the user will just does not know that an error has just occurred?